dm/ctl/master/start_task.go

```diff
@@ -48,6 +49,20 @@ func startTaskFunc(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
+	// If task's target db is configured with tls certificate related content
+	// the contents of the certificate need to be read and transferred to the dm-master
+	task := config.NewTaskConfig()
+	yamlErr := task.RawDecode(string(content))
+	if yamlErr != nil {
+		return yamlErr
+	}
+	if task.TargetDB != nil && task.TargetDB.Security != nil {
+		loadErr := task.TargetDB.Security.LoadTLSContent()
+		if loadErr != nil {
+			return loadErr
+		}
```

Comment:
It seems not compatible with the former version. If we still put TLS files on the dm-worker server we will fail to start a task now.

Reply:
Maybe I could add a hint to the error message to help the user locate the problem?

Reply:
Can we ignore this error here and let the dm-worker take a try?
If it still fails to read tls content, we can report an error. If users have several sources/tasks using tls before, it might be hard for them to change their way to use DM.

Reply:
Besides, I find that if we directly rolling upgrade DM from former version using MySQL tls, it seems that the former tasks will directly fail because they didn't have `SSLCABytes` in source config. Am I right?

Reply:
To ensure that tasks already using tls connections are not interrupted after this upgrade
Before the database connection is established, an attempt is made to read the configuration from the local disk

https://github.com/pingcap/dm/pull/1910/commits/6d62706696b724e168272dd7e7edcae00828cdab

cc @lance6716